tests/runtest.cmd

```diff
@@ -118,12 +119,6 @@ goto :eof
 set BUILDERRORLEVEL=%ERRORLEVEL%
 goto :eof
```

Comment:
That meesage actually points to the msbuild log which will only contain all possible failures , the html is sufficient for most people and is much easier to use to diagnose test failures